content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -509,88 +501,173 @@ kubelet [flags]
+BoundServiceAccountTokenVolume=true|false (ALPHA - default=false)<br/>
+CPUManager=true|false (BETA - default=true)<br/>
+CRIContainerLogRotation=true|false (BETA - default=true)<br/>
+CSIInlineVolume=true|false (BETA - default=true)<br/>
+CSIMigration=true|false (BETA - default=true)<br/>
+CSIMigrationAWS=true|false (BETA - default=false)<br/>
+CSIMigrationAWSComplete=true|false (ALPHA - default=false)<br/>
+CSIMigrationAzureDisk=true|false (BETA - default=false)<br/>
+CSIMigrationAzureDiskComplete=true|false (ALPHA - default=false)<br/>
+CSIMigrationAzureFile=true|false (ALPHA - default=false)<br/>
+CSIMigrationAzureFileComplete=true|false (ALPHA - default=false)<br/>
+CSIMigrationGCE=true|false (BETA - default=false)<br/>
+CSIMigrationGCEComplete=true|false (ALPHA - default=false)<br/>
+CSIMigrationOpenStack=true|false (BETA - default=false)<br/>
+CSIMigrationOpenStackComplete=true|false (ALPHA - default=false)<br/>
+CSIMigrationvSphere=true|false (BETA - default=false)<br/>
+CSIMigrationvSphereComplete=true|false (BETA - default=false)<br/>
+CSIStorageCapacity=true|false (ALPHA - default=false)<br/>
+CSIVolumeFSGroupPolicy=true|false (ALPHA - default=false)<br/>
+ConfigurableFSGroupPolicy=true|false (ALPHA - default=false)<br/>
+CustomCPUCFSQuotaPeriod=true|false (ALPHA - default=false)<br/>
+DefaultPodTopologySpread=true|false (ALPHA - default=false)<br/>
+DevicePlugins=true|false (BETA - default=true)<br/>
+DisableAcceleratorUsageMetrics=true|false (ALPHA - default=false)<br/>
+DynamicKubeletConfig=true|false (BETA - default=true)<br/>
+EndpointSlice=true|false (BETA - default=true)<br/>
+EndpointSliceProxying=true|false (BETA - default=true)<br/>
+EphemeralContainers=true|false (ALPHA - default=false)<br/>
+ExpandCSIVolumes=true|false (BETA - default=true)<br/>
+ExpandInUsePersistentVolumes=true|false (BETA - default=true)<br/>
+ExpandPersistentVolumes=true|false (BETA - default=true)<br/>
+ExperimentalHostUserNamespaceDefaulting=true|false (BETA - default=false)<br/>
+GenericEphemeralVolume=true|false (ALPHA - default=false)<br/>
+HPAScaleToZero=true|false (ALPHA - default=false)<br/>
+HugePageStorageMediumSize=true|false (BETA - default=true)<br/>
+HyperVContainer=true|false (ALPHA - default=false)<br/>
+IPv6DualStack=true|false (ALPHA - default=false)<br/>
+ImmutableEphemeralVolumes=true|false (BETA - default=true)<br/>
+KubeletPodResources=true|false (BETA - default=true)<br/>
+LegacyNodeRoleBehavior=true|false (BETA - default=true)<br/>
+LocalStorageCapacityIsolation=true|false (BETA - default=true)<br/>
+LocalStorageCapacityIsolationFSQuotaMonitoring=true|false (ALPHA - default=false)<br/>
+NodeDisruptionExclusion=true|false (BETA - default=true)<br/>
+NonPreemptingPriority=true|false (BETA - default=true)<br/>
+PodDisruptionBudget=true|false (BETA - default=true)<br/>
+PodOverhead=true|false (BETA - default=true)<br/>
+ProcMountType=true|false (ALPHA - default=false)br/>
```

Comment:
nit: Missing "<" for `<br/>` in ProcMountType

Reply:
how could I miss that ...
Thanks.